File: height-checker/height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        
        int[] actualHeight=new int[heights.length];
        for(int i=0;i<heights.length;i++)
            actualHeight[i]=heights[i];
        int count=0;
        Arrays.sort(heights);
        for(int i=0;i<heights.length;i++)
        {
            if(actualHeight[i]!=heights[i])
                count++;
        }
        return count;
    }
}

File: height-checker/height-checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        list1=list(heights)
        list1.sort()
        counter=0
        size=len(list1)
        for i in range(0,size):
            if heights[i]!=list1[i]:
                counter +=1
        return counter        
                